code/game/gamemodes/hivemind/hivemind.dm

```diff
@@ -36,6 +36,16 @@
 		return TRUE
 	return FALSE
 
+/proc/get_real_hivehost(mob/living/M) //Returns M unless it's under mind control, then it returns the original body
```

Comment:
why is this a global proc lol add it to some datum or something

Reply:
if you hate this proc don't look up

Reply:
@Kierany9 so put it on the mob living or on the ticker or literally anywhere else. 

This literally only operates on living mobs, and is only relevant to the hivemind gamemode

Reply:
Done